caffe2/experiments/python/SparseTransformer.py

```diff
@@ -106,13 +106,13 @@ def transFCRelu(cur, id2node, name2id, ops, model):
         cur.visited = True
         pre = cur
         flag = False
-        for _, temp in cur.ops.iteritems():
+        for _, temp in cur.ops.items():
```

Comment:
I considered this, but felt that since it is only an experiment script it was not necessary to add a compatibility module here. Do you disagree?